nomad/job_endpoint.go

```diff
@@ -114,8 +114,8 @@ func (j *Job) Register(args *structs.JobRegisterRequest, reply *structs.JobRegis
 	reply.Warnings = structs.MergeMultierrorWarnings(warnings...)
 
 	// Check job submission permissions
-	var aclObj *acl.ACL
-	if aclObj, err = j.srv.ResolveToken(args.AuthToken); err != nil {
+	aclObj, err := j.srv.ResolveToken(args.AuthToken)
+	if err != nil {
```

Comment:
Kind of cheating here to make the pattern closer to the other endpoints so it's easier to match them with `semgrep`, hope that's OK 😅

Reply:
Consistent looking code is one of Go's party tricks; I'm all for it :tada: